src/App.tsx

```diff
@@ -1,52 +1,37 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import {
+  Route,
+  Routes,
+  useNavigate,
+  useLocation,
+} from 'react-router-dom';
+import { TabsPage } from './pages/Tabs.page';
+import { HomePage } from './pages/Home.page';
+import { TabPage } from './pages/Tab.page';
+import { Page } from './components/layout/Page';
+import { NotFoundPage } from './pages/NotFound.page';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+export const App = () => {
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
 
-export const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
-        </div>
-      </div>
-    </nav>
+  if (pathname === '/home') {
+    navigate('/');
+  }
 
-    <div className="section">
-      <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="tabs is-boxed">
-          <ul>
-            <li data-cy="Tab" className="is-active">
-              <a href="#/">Tab 1</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 2</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Please select a tab
-        </div>
-      </div>
-    </div>
-  </>
-);
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<Page />}>
+          <Route index element={<HomePage />} />
+          <Route path="tabs" element={<TabsPage />}>
+            <Route path=":tabId" element={<TabPage />} />
+          </Route>
+        </Route>
+        <Route path="*" element={<NotFoundPage />} />
+      </Routes>
+    </>
+  );
```

Comment:
you have one parent element - so i don't think you need a fragment here
```suggestion
  return (
      <Routes>
        <Route path="/" element={<Page />}>
          <Route index element={<HomePage />} />
          <Route path="tabs" element={<TabsPage />}>
            <Route path=":tabId" element={<TabPage />} />
          </Route>
        </Route>
        <Route path="*" element={<NotFoundPage />} />
      </Routes>
  );
```

Reply:
As I remember it was giving an error but I'll check it out again.